Replace R2 with RZ in commands

9p4 opened the pull request:
Make rz-pm work for everybody.

9p4 committed:
Replace R2 with RZ in commands

ret2libc commented:
There are also to replace url like radare2-extras and similar. Also, that is not enough because of different API names so most of those plugins won't work just with this.

But it's definitely a start!

9p4 committed:
Rename + hyperlink (sketch af)

9p4 committed:
Fix some filenames

9p4 commented:
Issues so far:
https://github.com/radareorg/radare2-rlang
https://github.com/radareorg/radare2-bindings
https://github.com/radare/radare2-au
https://github.com/radareorg/r2dec-js
https://github.com/radareorg/radare2-sigs
https://github.com/securisec/r2msdn
https://github.com/nowsecure/r2lldb
https://github.com/leberus/r2k
https://github.com/thestr4ng3r/r2-os9.git
https://github.com/mrmacete/r2-ios-kernelcache
https://github.com/mrmacete/r2-ggpack
https://github.com/MaskRay/r2cLEMENCy
don't have their Rizin counterparts. I've guessed at what they should be when they are eventually added.

I've also decided to replace the instances of `r2` commands with `rizin` commands as `rz` conflicts with the `lrzsz` package.

There may have been some things that have slipped through, I'd recommend a full review before merging.

XVilka commented:
1) I think we can drop cLEMENCy since it was used just once in DEFCON CTF
2) I think we can also drop `radare2-au` - it is a toy after all.
3) I think we can drop `radare2-sigs` since signatures commands has changed anyway and script is very simple.
4) @wargio has changed the URL for `r2dec-js`
5) @thestr4ng3r can fix his `r2-os9`.
6) Native bindings can be removed since they have to be done from scratch anyway.
7) r2-ios-kernelcache support is now merged in radare2 itself (thus in rizin as well)
8) Regarding rlang plugins - I removed unsupported languages (at least for now), did renames and fixes for Python and put it into https://github.com/rizinorg/rz-lang

9p4 committed:
Cleanup + follow XVilka's recommendations

9p4 commented:
There's some URLs that are still under pancake's user repos that I haven't changed. I have dropped clemency, radare2-au, ghidra-rz-webserver, ghidra-rz-webserver, rz-au, rz-con, www-*, zig-masm32

I didn't see wargio's changed URL, and I've updated the rz-lang repo link.

Before removing a3a, do we need rust support?

wargio commented:
regarding r2dec, please change the URL to `rizinorg/jsdec` @sambhavsaggi

9p4 committed:
Update r2dec-js to jsdec

9p4 committed:
Cleanup + Fix some git URLs + updates

9p4 committed:
Fix spelling

XVilka commented:
`fs-squash` can be removed since we remove `fs` plugins from Rizin.

9p4 committed:
Remove fs-squash

ret2libc reviewed (approved):
I think this is okish... I'm a bit worried that this would give the feeling that plugins would work, when they actually don't because of the different API names. But maybe, as right now we don't have many different APIs a simple `sed` would work as well.

XVilka commented:
Please remove also `db/acr` since it's deprecated and out of the scope of Rizin package management anyway.

XVilka commented:
Please also remove `db/bokken`. Lol, didn't realize it was still there...

9p4 committed:
Remove ACR and bokken

ret2libc commented:
Merged! Thanks a lot. There is now a bit of work to do to fix the actual plugins for the renamed API. Most of them are likely very easy as of right now, however I would start with considering first the "most used" plugins.
